internal/cmds/cmds.go

```diff
@@ -388,14 +397,14 @@ func getOutput(ctx *log.Context, stdoutFileName string, stderrFileName string) (
 // checkAndSaveSeqNum checks if the given seqNum is already processed
 // according to the specified seqNumFile and if so, returns true,
 // otherwise saves the given seqNum into seqNumFile returns false.
-func checkAndSaveSeqNum(ctx log.Logger, seq int, mrseqPath string) (shouldExit bool, _ error) {
+func checkAndSaveSeqNum(ctx log.Logger, seq int, mrseqPath string, forceRerun bool) (shouldExit bool, _ error) {
 	ctx.Log("event", "comparing seqnum", "path", mrseqPath)
-	smaller, err := seqnum.IsSmallerThan(mrseqPath, seq)
+	isSavedSeqNumSmallerThanNewSeqNum, err := seqnum.IsSmallerThan(mrseqPath, seq)
 	if err != nil {
 		return false, errors.Wrap(err, "failed to check sequence number")
 	}
 
-	if !smaller {
+	if !forceRerun && !isSavedSeqNumSmallerThanNewSeqNum {
```

Comment:
What happens if the machine is rebooted? From this it appears we'll run the script every time RCV2 is executed. Why not just have CRP increment the seqNo?

Reply:
Good idea. I did think about it earlier. Do you mean instead of making this "ForceRun" (thinking to rename it) flag change in Windows and Linux extensions, we could just increment sequence number in CRP ? Only thing is it may not work right away if for some reason the VM has much higher sequence number than what CRP has. Especially, Linux uses files to note down mrseq number and if user reuses a disk for example. That may be an edge case though.

Reply:
I think just incrementing it in CRP is a better solution. I'll close this PR